homeassistant/components/logi_circle/camera.py

```diff
@@ -117,27 +124,6 @@ async def async_will_remove_from_hass(self) -> None:
         for detach in self._listeners:
             detach()
 
-    @property
-    def unique_id(self):
-        """Return a unique ID."""
-        return self._id
-
-    @property
-    def name(self):
-        """Return the name of this camera."""
-        return self._name
```

Comment:
Where was `self._name` set? As the name property takes precedence over the `attr_name` variable.

Reply:
Oh that PR is being shipped in the 2023.9. I'll create a new PR for this fix to include in the 2023.9